ui/src/pages/Upload.tsx

```diff
@@ -112,147 +158,230 @@ export const Upload: React.FC = () => {
+                  Enter where story was written
+                </StyledInputLabel>
+                <StyledSelect
+                  variant="outlined"
+                  value={formInput.current_city}
+                  onChange={handleChange}
+                  inputProps={{
+                    name: "current_city",
+                    id: "select-label-city",
+                  }}
+                >
+                  <MenuItem value={"Toronto"}>Toronto</MenuItem>
+                  <MenuItem value={"Calgary"}>Calgary</MenuItem>
+                  <MenuItem value={"Vancouver"}>Vancouver</MenuItem>
+                  <MenuItem value={"Thunder Bay"}>Thunder Bay</MenuItem>
+                </StyledSelect>
+              </FormControl>
+            </div>
+            <div>
+              <FormControl>
+                <UploadLabelsText>Year Published</UploadLabelsText>
+                <StyledSelect
+                  variant="outlined"
+                  value={formInput.year}
+                  onChange={handleChange}
+                  defaultValue={"2021"}
+                  inputProps={{
+                    name: "year",
+                    id: "select-label-year",
+                  }}
+                >
+                  <MenuItem value={"2021"}>2021</MenuItem>
+                  <MenuItem value={"2020"}>2020</MenuItem>
+                  <MenuItem value={"2019"}>2019</MenuItem>
+                </StyledSelect>
+              </FormControl>
+            </div>
+            <div>
+              <FormControl>
+                <UploadLabelsText>Tags</UploadLabelsText>
+                <StyledTags
+                  autoHighlight
+                  multiple
+                  id="tags-outlined"
+                  name="tags"
+                  freeSolo
+                  options={tagOptions ? tagOptions : [""]}
+                  filterSelectedOptions
+                  onChange={(event, newValue) => setTagArrayValues(newValue)}
+                  value={tagArray ? tagArray : [""]}
+                  renderTags={(value: string[], getTagProps) =>
+                    value.map((option: string, index: number) => (
+                      <Chip
+                        key={option}
+                        variant="outlined"
+                        label={option}
+                        {...getTagProps({ index })}
+                      />
+                    ))
+                  }
+                  renderInput={(params: TagParameters) => {
+                    params.inputProps.onKeyDown = handleKeyDown;
+                    return (
+                      <TextField
+                        {...params}
+                        variant="outlined"
+                        label="Shoe Tags"
+                        placeholder="Select Tags"
+                      ></TextField>
+                    );
+                  }}
+                />
+              </FormControl>
+            </div>
```

Comment:
We can probably replace the divs -> `<React.Fragment>` means we have less elements to render on the page